build-tools/packages/build-cli/src/base.ts

```diff
@@ -76,19 +79,21 @@ export abstract class BaseCommand<T extends typeof BaseCommand.flags> extends Co
     /**
      * @returns A default logger that can be passed to core functions enabling them to log using the command logging
      * system */
-    protected get logger(): Logger {
+    protected get logger(): CommandLogger {
         if (this._logger === undefined) {
             this._logger = {
                 info: (msg: string | Error) => {
                     this.log(msg.toString());
                 },
                 warning: this.warn.bind(this),
-                error: (msg: string | Error) => {
+                errorLog: (msg: string | Error) => {
```

Comment:
Should we rename the others to use the same naming convention (`warningLog`, etc.?)

Reply:
Also, my personal preference would be `logError` rather than `errorLog`, since it reads more like natural English. But don't block anything on this 😋